Resources/Locale/en-US/store/uplink-catalog.ftl

```diff
@@ -153,7 +153,7 @@ uplink-buldog-bundle-name = Bulldog Bundle
 uplink-buldog-bundle-desc = Lean and mean: Contains the popular Bulldog Shotgun, a 12g beanbag drum and three 12g buckshot drums.
 
 uplink-grenade-launcher-bundle-name = China-Lake Bundle
-uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher bundled with 8 rounds of various destruction capability.
+uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher bundled with 11 rounds of various destruction capability.
```

Comment:
Honestly I think putting hardcoded values into descriptions is silly and we should just remove the number entirely.

Reply:
I'd do that if I could but I wouldn't have any idea how to do that unfortunately. I can stop fixing values if there is a plan to fix it permanently.